erin_averill/forLoop_basicI.py

```diff
@@ -0,0 +1,40 @@
+# Basic - Print all the numbers/integers from 0 to 150.
+def print():
+	for number in range(1, 151):
+	    print(number)
+
+# Multiples of Five - Print all the multiples of 5 from 5 to 1,000,000.
+def multiple():
+	for number in range(5,1000001):
+	    if number %5 == 0:
+	        print(number)
+
+# Counting, the Dojo Way - Print integers 1 to 100.  If divisible by 5, print "Coding" instead. 
+# If by 10, also print " Dojo".
+def counting():
+	for number in range(1, 100):
+	    if number %10 == 5:
+	        print('Coding')
+	    elif number %10 == 0:
```

Comment:
for numbers like 10,20,30...it should print 'coding dojo'. Fizz buzz(similar to this one) is a famous interview question. Think about it!